CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # OverReact Changelog
```

Comment:
TODO also add 
- improved messaging for runtime errors when state is missing
- build time error/warning when state is missing?

Reply:
FYI @joebingham-wk I plan to address these in this PR